tests/printer/printer.cpp

```diff
@@ -294,6 +294,40 @@ TEST(Printer, printModelWithStandardUnitsAdded)
     EXPECT_EQ(e, printer->printModel(model));
 }
 
+TEST(Printer, printModelImportingModelParentComponent)
+{
+    auto parser = libcellml::Parser::create();
+    auto modelContents = fileContents("importingModelParentComponent.cellml");
+    auto model = parser->parseModel(modelContents);
+    EXPECT_EQ(size_t(0), parser->errorCount());
```

Comment:
Any particular reason why it's `errorCount()` and not `issueCount()`?

Reply:
In general we should be using what we want to be checking.  I think there's even an open issue for exactly that.  As the Issue class becomes more widely used, we'll start to see information items stored in the logger, so it's better to check for errors when you mean to check for errors, and issues when you mean to check for anything.